### ISSUE TITLE
Exclude doesn't work recursively

### ISSUE THREAD

**Issue by bee-keeper** [bruth/django-preserialize#12]
Hello,

It seems that exclude has no effect on nested serialised objects.  Is this intentional?

**Comment by bruth** [comment]
Can you provide an example?

**Comment by bee-keeper** [comment]
I have a Django model with a fk and m2m.  I run the following, the field is only excluded in the outer level and appears in all subsequent nestings.  I haven't had too much time to look into this, perhaps using templates is the solution? 

`serialize(my_obj, exclude=['my_field'])`

**Comment by bruth** [comment]
@bee-keeper Sorry for the late response. For recursive objects or models that appear more than one in a graph, templates are very good for. Here is [an example](https://github.com/chop-dbhi/serrano/blob/master/serrano/resources/templates.py) of how I define and reference them.

**Comment by bee-keeper** [comment]
Ok i had assumed that templates would be a solution, I suppose it was more a comment on the default behaviour without using a template.  Please feel free to close in that case.

**Comment by bruth** [comment]
> I suppose it was more a comment on the default behaviour without using a template.

Yes. That being said, #5 describes an approach to define defaults per model. Also I have #14 in the works that defines a `Serializer` class that would encapsulate options like these.
